shared/types/proto/events.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+import "google/protobuf/any.proto";
+
+package pocket;
+
+option go_package = "/types/";
+
+enum SourceModule {
```

Comment:
I think it's more clean to name this something like: `Concern' or 'Scope' than 'Module'. Since a naming like "module" is somewhat meta (_as in meta-programming_) and reverts the attention off of the actual business of the code into the code structure itself.

I think `SCOPE` is more fitting.

Reply:
Removing altogether until we need it again.